tests/ui_tools/test_buttons.py

```diff
@@ -10,10 +10,10 @@
 from zulipterminal.config.symbols import CHECK_MARK, MUTE_MARKER
 from zulipterminal.ui_tools.buttons import (
     DecodedStream,
+    DMButton,
```

Comment:
Additionally, avoid using separate commits solely to fix test issues. Every test must pass on each commit since the `Linting & tests / Ensure isolated PR commits (pull_request)` check ensures that all commits stand alone successfully.

Maybe try rebasing to correct issues in each commit as needed. 

For example:

In the commit `Refactor: replace 'PM' with 'DM' (ignoring model.py, test_model.py, f…)`, isort is not satisfied (which you fixed in this commit separately). Rebase, address this, and make sure each commit individually passes all tests.

Reply:
Got it! I just thought some clarification might be helpful since I assumed you were new to this. My mistake if I jumped in too early!

Good luck :)